aim/cli/watcher_cli.py

```diff
@@ -0,0 +1,171 @@
+    check_configuration(repo)
+
+    ctx.obj['config'] = get_config(repo.path)
+    if ctx.invoked_subcommand is None:
+        ctx.invoke(add_config)
+
+
+@config.command('dump')
+@click.pass_context
+def dump_config(ctx):
+    config = ctx.obj['config']
+    click.echo(config.dump())
+
+
+@config.group('add', invoke_without_command=True)
+@click.pass_context
+def add_config(ctx):
+    if ctx.invoked_subcommand is None:
+        add_new = True
+        while add_new:
+            sub_commands = ctx.command.list_commands(ctx)
+            choice = click.prompt('Select notifier type to add:', show_choices=True, type=click.Choice(sub_commands))
+            sub_cmd = ctx.command.get_command(ctx, choice)
+            extra_args = {}
+            for param in sub_cmd.params:
+                if isinstance(param, click.Option) and param.prompt:
+                    extra_args[param.name] = param.prompt_for_value(ctx)
+            ctx.invoke(sub_cmd, **extra_args)
+            add_new = click.confirm('Would you like to add another Notifier?')
+
+
+@config.command('remove')
+@click.argument('notifier-id', required=True, type=str)
+@click.pass_context
+def remove_config(ctx, notifier_id):
+    cfg = ctx.obj['config']
+    if notifier_id in cfg.notifiers:
+        click.echo(json.dumps(cfg.get(notifier_id), indent=2))
+        click.confirm('Remove configuration above?', abort=True)
+        cfg.remove(notifier_id)
+        cfg.save()
+    else:
+        click.echo(f'No notifier with id {notifier_id} found.')
+
+
+@config.command('enable')
+@click.argument('notifier-id', required=True, type=str)
+@click.pass_context
+def enable_config(ctx, notifier_id):
+    cfg = ctx.obj['config']
+    if notifier_id in cfg.notifiers:
+        cfg.enable(notifier_id)
+        cfg.save()
+    else:
+        click.echo(f'No notifier with id {notifier_id} found.')
+
+
+@config.command('disable')
+@click.argument('notifier-id', required=True, type=str)
+@click.pass_context
+def disable_config(ctx, notifier_id):
+    cfg = ctx.obj['config']
+    if notifier_id in cfg.notifiers:
+        cfg.disable(notifier_id)
+        cfg.save()
+    else:
+        click.echo(f'No notifier with id {notifier_id} found.')
+
+
+@add_config.command('workplace')
+@click.option('--group-id', prompt=True, required=True, type=int)
+@click.option('--access-token', prompt=True, required=True, type=str)
+@click.option('--message', prompt=True, required=False, type=str, default=DEFAULT_MESSAGE_TEMPLATE, show_default=True)
+@click.pass_context
+def workplace_config(ctx, group_id, access_token, message):
+    cfg = ctx.obj['config']
+    new_cfg = {
+        'id': str(uuid.uuid4()),
+        'type': 'workplace',
+        'arguments': {
+            'group_id': group_id,
+            'access_token': access_token,
+            'message': message,
+        }
+    }
+    click.echo(json.dumps(new_cfg, indent=2))
+    click.confirm('Save configuration above?', abort=True)
```

Comment:
<img width="780" alt="image" src="https://user-images.githubusercontent.com/13848158/183980401-55904be6-109a-49d2-9198-4f3f3157ba10.png">

the json config looks a bit weird. what about dislaying formatted version like:

```
Type: python logger
Message: ""
------
Save the config above? y/n
```

Reply:
<img width="376" alt="image" src="https://user-images.githubusercontent.com/13848158/183980668-88598a5d-5310-434c-8bdb-dd9d0a58950a.png">

"notifier" sounds better, let's use it everywhere. so instead of 
"Repo /Users/gor/Projects/aim/aim/.aim has no configured notifications. "
display 
"Repo /Users/gor/Projects/aim/aim/.aim has no configured notifiers."  (with lower case?)